controller/workitem.go

```diff
@@ -255,20 +250,32 @@ func (c *WorkitemController) Delete(ctx *app.DeleteWorkitemContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, err)
```

Comment:
I can't edit or delete previous comments :disappointed:

Reply:
>Why do you want to throw an error for 'deletion' when actually the error happens because of failure to load the workitem?

Because the action was to `DELETE`. An error like `failed to delete workitem: Failed to list workitem...` would seem more informative. I am suggesting we wrap the error and then return it 

```go
 return jsonapi.JSONErrorResponse(ctx, errs.Wrap("failed to delete workitem", err))
```

Reply:
@jarifibrahim understand your point. Addressed it here e8a08a5